package.json

```diff
@@ -53,6 +54,10 @@
   "homepage": "https://github.com/datagovsg/postmangovsg#readme",
   "devDependencies": {
     "concurrently": "^7.0.0",
+    "cypress": "^9.6.1",
```

Comment:
I'm thinking whether we should have cypress tests in its own module (something like what we did with `backend`, `frontend`, etc) since it's a standalone test suite

Reply:
rearranged